src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementIncidentProvider.java

```diff
@@ -0,0 +1,79 @@
+package fi.otavanopisto.kuntaapi.server.integrations.management;
+
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+
+import fi.metatavu.kuntaapi.server.rest.model.Incident;
+import fi.otavanopisto.kuntaapi.server.controllers.IdentifierRelationController;
+import fi.otavanopisto.kuntaapi.server.id.IncidentId;
+import fi.otavanopisto.kuntaapi.server.id.OrganizationId;
+import fi.otavanopisto.kuntaapi.server.integrations.IncidentProvider;
+import fi.otavanopisto.kuntaapi.server.integrations.management.resources.ManagementIncidentResourceContainer;
+import javax.enterprise.context.ApplicationScoped;
+
+/**
+ * Incident provider for management wordpress
+ * 
+ * @author Antti Leppä
+ */
+@ApplicationScoped
+@SuppressWarnings ("squid:S3306")
+public class ManagementIncidentProvider extends AbstractManagementProvider implements IncidentProvider {
+  
+  @Inject
+  private ManagementIncidentResourceContainer managementIncidentCache;
+  
+  @Inject
+  private IdentifierRelationController identifierRelationController;
+  
+  @Override
+  public List<Incident> listOrganizationIncidents(OrganizationId organizationId, String slug, OffsetDateTime startBefore, OffsetDateTime endAfter) {
+    List<IncidentId> incidentIds = identifierRelationController.listIncidentIdsBySourceAndParentId(ManagementConsts.IDENTIFIER_NAME, organizationId);
+    List<Incident> incidents = new ArrayList<>(incidentIds.size());
+    
+    for (IncidentId incidentId : incidentIds) {
+      Incident incident = managementIncidentCache.get(incidentId);
+      System.out.println("-------------------------");
+      if (incident != null && isAcceptable(incident, slug, startBefore, endAfter)) {
+        incidents.add(incident);
+      } else {
+        System.out.println("No match: " + incident.getId());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "incident" is nullable here [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2259)